stock_packaging_usability_pp/wizard/stock_select_product_packaging.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2014-2021 Akretion France (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class StockSelectProductPackaging(models.TransientModel):
+    _name = "stock.select.product.packaging"
+    _description = "Select Packaging"
+
+    packaging_id = fields.Many2one("product.packaging", string="Packaging")
+    # required=False, because we accept that it can be left empty
+
+    def validate(self):
+        self.ensure_one()
+        default_packaging_id = self.packaging_id.id or False
+        assert (
```

Comment:
@alexis-via We are in an Odoo context and not in a pure python environment.

IMHO it's better to use UserError. Same for all